SimGeneral/PreMixingModule/python/mixOne_premix_on_sim_cfi.py

```diff
@@ -199,10 +163,39 @@
             accumulator = _recoTrackAccumulator.clone(
                 pileUpTracks = "mix:generalTracks"
             )
-        )
+        ),
+        dt = dict(
+            labelSig = "mix:MuonSimHitsMuonDTHits",
+            pileInputTag = "mix:MuonSimHitsMuonDTHits",
+            collectionDM = "MuonSimHitsMuonDTHits",
+        ),
+        rpc = dict(
+            labelSig = "mix:MuonSimHitsMuonRPCHits",
+            pileInputTag = "mix:MuonSimHitsMuonRPCHits",
+            collectionDM = "MuonSimHitsMuonRPCHits",
+        ),
+        csc = dict(
+            labelSig = "mix:MuonSimHitsMuonRPCHits",
+            pileInputTag = "mix:MuonSimHitsMuonCSCHits",
+            collectionDM = "MuonSimHitsMuonCSCHits",
+        ),
     ),
 )
 
+from Configuration.Eras.Modifier_run2_GEM_2017_cff import run2_GEM_2017
+from Configuration.Eras.Modifier_run3_GEM_cff import run3_GEM
+(run2_GEM_2017 | run3_GEM).toModify(
+    mixData,
```

Comment:
it looks like that this update is the main reason the GEMs now show up in the mixed outputs in amount more consistent with the classic mixing.

Reply:
Correct, I'll add that to the PR description (thanks for pointing it out).